vendor.conf

```diff
@@ -2,7 +2,7 @@
 github.com/urfave/cli v1.20.0
 github.com/kr/pretty v0.1.0
 github.com/kr/text v0.1.0
-github.com/containers/image ff926d3c79684793a2135666a2cb738f44ba33dc
+github.com/containers/image 2d92e2533d517b6db4ae7d085a332915fec57867
```

Comment:
That commit does not actually exist yet, which would make `make vendor` not reproducible.  After https://github.com/containers/image/pull/631 is merged, please update this with the actual merge commit and re-vendor.

Reply:
done